SPECS/abseil-cpp/abseil-cpp.spec

```diff
@@ -68,15 +72,15 @@ Development headers for %{name}
   -DABSL_ENABLE_INSTALL:BOOL=ON \
   -DABSL_BUILD_TESTING:BOOL=ON \
   -DABSL_BUILD_TEST_HELPERS:BOOL=ON \
-  -DCMAKE_BUILD_TYPE:STRING=None \
+  -DCMAKE_BUILD_TYPE:STRING=RelWithDebInfo \
   -DCMAKE_CXX_STANDARD:STRING=17
 %cmake_build
 
 %install
 %cmake_install
 
 %check
-%ctest --output-on-failure
+%ctest --output-on-failure --exclude-regex waiter_test
```

Comment:
Just copy-pasting your changelog entry, so future generations have it easier to understand why this one test got disabled:
```suggestion
# Disable flaky waiter tests, which sleep on the monotonic timer and then,
# inexplicably, test how much time has passed against the system timer.
%ctest --output-on-failure --exclude-regex waiter_test
```

Btw., there wouldn't be a bug/issue opened upstream for this, so that we could link it here, would there?

Reply:
I haven't opened a bug upstream as of yet, but [here's evidence that the Fedora `grpc` maintainer __knows__ there's a problem](https://src.fedoraproject.org/rpms/grpc/pull-request/35).

Before I file a PR, I intend to compile and test the solution in a Fedora `mock` chroot to make absolutely sure `abseil-cpp` is compiled with `NDEBUG` defined with the same changes.  I intended to use the `mock` version in Azure Linux to do the tests; unfortunately, many of the Fedora-specific `mock` configs in Azure Linux's `mock` are old and don't work anymore, and I haven't gotten around to either updating `mock` in Azure Linux or creating a WSL instance with a more recent `mock`.